### PR TITLE
added.. a basic style  div's etc 

### PR DESCRIPTION
**Pull request by A-Lord**
and made tailwind components., also changed the side bar so it now has break points instead of a button.

### CODE REVIEW

**Review by Omar-Slik — APPROVED** [review]

**Review by AhmadALshalabi — COMMENTED** [review]
visit:  https://animate.style/
-------------------------------
steps to put animation on your page: 
1- npm install animate.css --save
2- import "animate.css"; //to your component
3- use "transition" buildin animation element 
4- use the class="enter-active-class="" // for "transition" 
5- add the desired animation style from the webpage 
6- enjoy!

test current animation > npm i

File: src/components/UI/ThemeToggle.vue [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
-      v-if="theme === 'light-theme'"
-      icon="moon"
-      class="text-xl text-[#fff] hover:text-[#2b2b2b]"
-    />
-    <font-awesome-icon v-else icon="sun" class="text-xl text-[#ccf807]" />
-  </button>
+	<div @click="ToggleTheme" class="toggle">
+		<font-awesome-icon v-if="theme === 'light-theme'" icon="moon" class="text-xl text-[#fff] hover:text-[#2b2b2b]" />
+		<font-awesome-icon v-else icon="sun" class="text-xl text-[#ccf807]" />
+		<span class="sideBarText"> Theme </span>
```
Comment (AhmadALshalabi) [inline src/components/UI/ThemeToggle.vue:5]: tested without "Theme" word  and looks better

**Comment by A-Lord** [comment]
Vi har ju redan animationer https://tailwindcss.com/docs/transition-property  =)

**Comment by AhmadALshalabi** [comment]
> Vi har ju redan animationer https://tailwindcss.com/docs/transition-property =)

way eaiser to use the commented one :)

**Comment by A-Lord** [comment]
> > Vi har ju redan animationer https://tailwindcss.com/docs/transition-property =)
> 
> way eaiser to use the commented one :)

kanske ., använd https://jofftiquez.github.io/v-animate-css/ .. för vue verisionen,.., men less is more när det gäller animationer för web.  Kollade och var inga av dom nya animationerna som passa på en budget sida. för flashiga :P

File: src/assets/tailwind.css [hunk 2]
```diff
@@ -4,51 +4,108 @@
...
+    focus:invalid:border-red-500 focus:invalid:ring-red-500 text-textcolor-input;
+	}
+	button {
+		@apply px-4 py-1.5 rounded-md shadow-lg bg-sky-800 font-medium text-gray-100 block hover:bg-sky-700 transition duration-300 disabled:bg-background-secondary;
+	}
+	form {
+		@apply flex flex-col justify-center;
+	}
+	textarea {
+		@apply mb-3 px-2 py-1.5 mb-3 mt-1 block w-full px-2 py-1.5 border border-gray-300 rounded-md text-sm shadow-sm placeholder-gray-400
```
Comment (AhmadALshalabi) [inline src/assets/tailwind.css]: double px, py (row 12) 
double px, py, mb (row 27 & 36)
dont know if this will cause problem with tailwind yet!

Comment (AhmadALshalabi) [reply src/assets/tailwind.css]: fixed no worry

**Review by AhmadALshalabi — APPROVED** [review]

**Review by Omar-Slik — APPROVED** [review]

**Review by KimNordstrom — APPROVED** [review]

**Pull request merged**